packages/env/lib/cli.js

```diff
@@ -33,7 +33,12 @@ const withSpinner = ( command ) => ( ...args ) => {
 				) }ms)`
 			);
 		},
-		( err ) => spinner.fail( err.message || err.err )
+		( err ) => {
+			spinner.fail( err.message || err.err );
+			// eslint-disable-next-line no-console
+			console.error( `\n\n${ err.out || err.err }\n\n` );
```

Comment:
Should we prefer the `stdout` ? Seems like the error message might have more useful contents. Or both, even.

Reply:
Shouldn't we send errors to `stderr`? To avoid unexpected outputs being piped to files and all that?

Reply:
But if we're handling these errors, I would expect they're not going to be logged to `stderr` by default?

Reply:
They're not, that's why I log them here.

Reply:
> They're not, that's why I log them here.

But if there's both `stdout` and `stderr`, with this implementation we'd only be logging the former and not the latter?

Reply:
`err.out` and `err.err` are both error messages. It's just that some of the used libs format the error object thrown differently.